scripts/steps/build-api.js

```diff
@@ -110,6 +104,105 @@ const UNWRAP_UTILITIES = new Set([
   'Readonly'
 ])
 
+const EXCLUDED_SUBMENU_KINDS = new Set(['Type alias', 'Interface', 'Namespace'])
+const EXCLUDED_TREE_KINDS = new Set(['Namespace'])
+const CORE_ENTITIES = new Set([
+  'BaseNode',
+  'ClientNode',
+  'Connection',
+  'Log',
+  'LogStore',
+  'Reconnect',
+  'ServerConnection',
+  'ServerNode'
+])
+const CLIENT_ENTITIES = new Set([
```

Comment:
Add `IndexedStore` since it is important feature (should be discoverable from submenu).

If you want you can put `MemoryStore` to `Core` for consistency.

Reply:
Add defineAction. It is useful action creator.

Reply:
Add `log()`, `favicon()`, `attention()`, useful UX extension (like change favicon on disconnection)

Reply:
Сan I place `defineAction` in the `Core` group since this function is present in both node and web API pages, but `Client` group is missing in node api page?

Reply:
Yeap, good idea